code/modules/client/loadout/loadout_suit.dm

```diff
@@ -102,11 +102,6 @@
 	display_name = "labcoat"
 	path = /obj/item/clothing/suit/toggle/labcoat
 
-/datum/gear/suit/raincoat
-	display_name = "Cybersun labcoat"
```

Comment:
Can you just rename this/and the object? This thing was named Cybersun before we even had lore for them

Reply:
Does renaming it to "Transparent Labcoat" work?

Reply:
sure, if it's used on cybersun ships... well, tbh cybersun should have a new sprite for it.